builtin/game/register.lua

```diff
@@ -338,25 +398,24 @@ function core.run_callbacks(callbacks, mode, ...)
 	if cb_len == 0 then
 		if mode == 2 or mode == 3 then
 			return true
-		elseif mode == 4 or mode == 5 then
+		end
```

Comment:
but it makes it easier to understand that the function doesn't continue after the return